lib/service/postcard_service.dart

```diff
@@ -274,16 +274,18 @@ class PostcardServiceImpl extends PostcardService {
          lat: $lat, lon: $lon, counter: $counter
         ''');
       final result = await _postcardApi.updatePostcard(
-          tokenId: tokenId,
-          data: image,
-          metadata: metadata,
-          signature: signature,
-          address: address,
-          publicKey: publicKey,
-          lat: lat,
-          lon: lon,
-          counter: counter,
-          promptID: counter > 1 ? null : prompt?.id) as Map<String, dynamic>;
+        tokenId: tokenId,
+        data: image,
+        metadata: metadata,
+        signature: signature,
+        address: address,
+        publicKey: publicKey,
+        lat: lat,
+        lon: lon,
+        counter: counter,
+        promptID: counter > 1 ? null : prompt?.id,
+        prompt: counter > 1 ? null : prompt?.description,
```

Comment:
please use `promtDescription` (or something like this) to avoid confusing.

Reply:
use "promtDescription" doesn't make it better because it doesn't match with api unless we want to fix api too